### PR TITLE
pd: use 10MB as mempool limit in comet config template

### PR DESCRIPTION
**Pull request by dynst**
## Describe your changes

Lower the default value for `max_txs_bytes` from 1GiB to 10MiB. Upstream also substantially lowered this default recently. 
https://github.com/cometbft/cometbft/pull/2767

See also the [new `nop` mempool type](https://github.com/cometbft/cometbft/pull/1643) they added to deal with the mempool being a potential DoS vector a la [p2p-storms](https://github.com/notional-labs/placid).

## Checklist before requesting a review

- [X] If this code contains consensus-breaking changes, I have added the "consensus-breaking" label. Otherwise, I declare my belief that there are not consensus-breaking changes, for the following reason:

The mempool and its configuration are outside consensus.

### CODE REVIEW

File: testnets/cometbft_config_template.toml [hunk 1]
```diff
@@ -290,10 +290,11 @@ wal_dir = ""
...
 
-# Limit the total size of all txs in the mempool.
-# This only accounts for raw transactions (e.g. given 1MB transactions and
-# max_txs_bytes=5MB, mempool will only accept 5 transactions).
-max_txs_bytes = 1073741824
+# The maximum size in bytes of all transactions stored in the mempool.
+# This is the raw, total transaction size. For example, given 1MB
+# transactions and a 5MB maximum mempool byte size, the mempool will
+# only accept five transactions.
+max_txs_bytes = 67108864
```
Comment (erwanor) [inline testnets/cometbft_config_template.toml:297]:
```suggestion
max_txs_bytes = 5242880
```

About x5 the block size, this is ultimately a user controlled parameter

Comment (dynst) [reply testnets/cometbft_config_template.toml]: For posterity's sake: the block size is being set to 1MiB in #4597.

**Comment by erwanor** [comment]
Increased it to 10MB to be more coherent with maximum number of individual transactions that the template specifies (5000, at 2KB each ~ 10MB)

**Comment by erwanor** [comment]
Once again, thanks a lot, this is really helpful!

**Review by erwanor — APPROVED** [review]

**Comment by dynst** [comment]
> `   ERROR penumbra_view::storage: failed to connect to app query service endpoint, error: tonic::transport::Error(Transport, hyper::Error(Connect, ConnectError("tcp connect error", Os { code: 111, kind: ConnectionRefused, message: "Connection refused" })))`
>
> `test view_server_can_be_served_on_localhost ... FAILED`

This was fine with 64MiB, but breaks with 10MiB?

**Pull request merged**